examples/github-demo.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: brigade.sh/v2
+kind: Project
+metadata:
+  id: github-demo
+description: A project that demonstrates integration with GitHub
+spec:
+  workerTemplate:
+    git:
+      cloneURL: https://github.com/krancour/brigade2-pipeline-demo.git
```

Comment:
Let's remove this new example.

In another recent commit already merged into `v2`, the `examples/` directory was restructured.

This example already exists in the `v2` branch, albeit at a different path.

Again-- just trying to cut through some noise.

Reply:
Removed.